package.json

```diff
@@ -8,7 +8,7 @@
     "preinstall": "rm -rf ./public/uv",
     "postinstall": "yarn run uv-install && yarn run uv-config",
     "uv-install": "shx cp -r ./node_modules/universalviewer/dist ./public/uv",
-    "uv-config": "shx cp ./config/uv/uv.html ./public/uv/uv.html & shx cp ./config/uv/uv-config.json ./public/uv/ & shx cp ./config/uv/uv.js ./public/uv/uv.js"
+    "uv-config": "shx cp ./config/uv/uv.html ./public/uv/uv.html & shx cp ./config/uv/uv-config.json ./public/uv/ & shx cp ./config/uv/uv.js ./public/uv/uv.js & shx cp ./config/uv/uv-en-uofl-theme/ ./public/uv/themes/uv-en-uofl-theme/"
```

Comment:
Thank you @alishaevn

Reply:
:)

Reply:
I had to make the cp command recursive to get around the "cp: omitting directory `./config/uv/uv-en-uofl-theme/`" error

Reply:
Yes this will help us get that copied over automatically! Thank you @labradford & @alishaevn & @rkuehn-uofl 🥳